fastapi/security/open_id_connect_url.py

```diff
@@ -12,6 +12,13 @@ class OpenIdConnect(SecurityBase):
     """
     OpenID Connect authentication class. An instance of it would be used as a
     dependency.
+
+    Note:
+    This class **does not perform any token validation or decoding**.
+    It only extracts the `Authorization` header and includes metadata in the OpenAPI docs.
+
+    You must implement the actual authentication logic separately (e.g., verifying
+    the token signature, claims, and user handling).
```

Comment:
```suggestion
        You must handle actual token validation separately (e.g. signature and claims verification).
```
Removed empty line, added indentation, removed ", and user handling" to make text unified across classes